appveyor.yml

```diff
@@ -4,10 +4,6 @@ install:
 - ps: (new-object Net.WebClient).DownloadString("https://raw.github.com/madskristensen/ExtensionScripts/master/AppVeyor/vsix.ps1") | iex
 - powershell -NoProfile -ExecutionPolicy unrestricted -Command "&{iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/dotnet/cli/rel/1.0.1/scripts/obtain/dotnet-install.ps1'))}"
 
-before_build:
-  - ps: Vsix-IncrementVsixVersion | Vsix-UpdateBuildVersion
-  - ps: Vsix-TokenReplacement src\LibraryManager.Vsix\source.extension.cs 'Version = "([0-9\\.]+)"' 'Version = "{version}"'
-
 build_script:
   - dotnet restore LibraryManager.sln -v quiet
```

Comment:
Can you replace this line with "msbuild /t:restore /v:m" as you suggested? I tried locally and it works.